UnityProject/Assets/Scripts/Objects/Medical/CloningConsole.cs

```diff
@@ -158,12 +142,22 @@ public void ServerTryClone(CloningRecord record)
 		{
 			if (cloningPod && cloningPod.CanClone())
 			{
-				var status = record.mind.GetCloneableStatus(record.mobID);
+				Mind mind = NetworkUtils.FindObjectOrNull(record.mindID)?.GetComponent<Mind>();
```

Comment:
there's an extension to uint to get the game object/networkk identity
NetIdToGameObject();
NetworkIdentity();

Reply:
this is fine